src/gui/MainWindow.cpp

```diff
@@ -822,7 +818,11 @@ void MainWindow::onActiveDocumentChanged(const int newIndex){
             for(QAction * action:singleViewAct) action->setChecked(false);
             singleViewAct[documents[activeDocumentId]->getDisplayGrid()->getActiveDisplayId()]->setChecked(true);
         }
-
+    }else if (displayGrid == nullptr && activeDocumentId != -1) {
```

Comment:
That's logically right, but I don't like the duplicated test of displayGrid. The expressions in the if-clauses give the impression that something complicated is going on there. I recommend to simplify it to
```
if (displayGrid != nullptr) {
    if (displayGrid->getDocument()->getDocumentId() != activeDocumentId) {
        ~
    }
else if (activeDocumentId != -1) {
    ~
}
```
Logically both solutions are the same, but the above describes better that first, the document type is tested (displayGrid != nullptr: BRL-CAD database or something else, i.e. the help page) and then, if something has changed there (another database, activeDocumentId not reset yet), the dependent views will be changed too.

Reply:
So `activeDocumentId != -1` is enough to check if the tab is `Quick Start` or not.